qemu/tests/virtio_port_hotplug.py

```diff
@@ -28,35 +29,46 @@ def run(test, params, env):
     :param params: Dictionary with the test parameters.
     :param env:    Dictionary with test environment.
     """
-
     vm = env.get_vm(params["main_vm"])
     vm.verify_alive()
+    os_type = params["os_type"]
     timeout = int(params.get("login_timeout", 360))
-    if params["os_type"] == "windows":
-        run_bg_test(test, params, vm)
-    for repeat in range(int(params.get("repeat_times", 1))):
-        repeat += 1
+    module = params.get("modprobe_module")
+    check_module = params.get("check_module", "yes")
```

Comment:
@nanliu-r get_boolean, seems you miss the pre comment :)

Reply:
ok